packages/web-opds-client/src/__tests__/DataFetcher-test.ts

```diff
@@ -17,9 +17,9 @@ describe("DataFetcher", () => {
     });
 
     it("uses fetch()", () => {
-      let options = {
+      let options: RequestInit = {
         method: "POST",
-        data: { test: "test" },
+        body: "test",
```

Comment:
Data isn't a valid `RequestInit` option, so I had to change it once we were enforcing types. 

https://developer.mozilla.org/en-US/docs/Web/API/RequestInit